bankgreen/urls.py

```diff
@@ -53,4 +54,29 @@
     path(
         "check_prismic_mismatches/", views.check_prismic_mismatches, name="check_prismic_mismatches"
     ),
+    path(
+        "password_reset/",
+        CustomPasswordResetView.as_view(
+            html_email_template_name="registration/custom_password_reset_email.html",
+            subject_template_name="registration/custom_password_reset_subject.txt",
+        ),
+        name="password_reset",
+    ),
+    path(
+        "password_reset/done/",
+        auth_views.PasswordResetDoneView.as_view(),
+        name="password_reset_done",
+    ),
+    path(
+        r"^reset/<uidb64>/<token>/",
```

Comment:
I am getting a warning in Django here. I think you're not supposed to start the path with "^"
![image](https://github.com/bank-green/bankgreen-django/assets/68507104/ad262c40-2575-48c8-aa8c-291d37031a16)

Reply:
Updated urls.py file. I don't see warning message now.